doc/git_workflow.md

```diff
@@ -0,0 +1,41 @@
+# Git Workflow for SQLFlow project
+This doc is mainly written for new contributors in this project, those who are experienced open source developers can skip the doc. Please read carefully and leave comments as it needs and helps us improve as more people from the community are joing force. 
+
+## Fetch Upstream
+First let's run git fetch to download commits, files, and refs from a remote repository into your local repo
+```bash
+git fetch upstream develop
```

Comment:
You may need to clarify `upstream` and `origin`. For example, if one `go get github.com/sql-machine-learning/sqlflow`. In his local repo, the `origin` refers to the official repo https://github.com/sql-machine-learning/sqlflow, instead of his fork.

Since our `build.md` recommends `go get` to download the source code, I would suggest adjusting `upstream` and `origin` accordingly. Also, adding a sample of `.git/config` will also help. Here is mine:
```
$ cat .git/config
[core]
	repositoryformatversion = 0
	filemode = true
	bare = false
	logallrefupdates = true
	ignorecase = true
	precomposeunicode = true
[remote "origin"]
	url = https://github.com/sql-machine-learning/sqlflow
	fetch = +refs/heads/*:refs/remotes/origin/*
	fetch = +refs/pull/*/head:refs/remotes/pr/*
[remote "tonyyang-svail"]
	url = https://github.com/tonyyang-svail/sqlflow
	fetch = +refs/heads/*:refs/remotes/origin/*
	fetch = +refs/pull/*/head:refs/remotes/pr/*
```

Reply:
Sure, will use your git config as an example here. Sounds good?

Reply:
Resolved and committed in the PR, please review.